specs/compliance/aws-eks-cis-1.4.yaml

```diff
@@ -0,0 +1,381 @@
+      objects and actions that can be taken on those objects. It is possible to set either of
+      these to be the wildcard "*" which matches all items.
+      Use of wildcards is not optimal from a security perspective as it may allow for
+      inadvertent access to be granted when new resources are added to the Kubernetes API
+      either as CRDs or in later versions of the product.
+    checks:
+      - id: AVD-KSV-0044
+      - id: AVD-KSV-0045
+      - id: AVD-KSV-0046
+    severity: HIGH
+  - id: 4.1.4
+    name: Minimize access to create pods (Manual)
+    description: |
+      The ability to create pods in a namespace can provide a number of opportunities for
+      privilege escalation, such as assigning privileged service accounts to these pods or
+      mounting hostPaths with access to sensitive data (unless Pod Security Policies are
+      implemented to restrict this access)
+      As such, access to create new pods should be restricted to the smallest possible group
+      of users.
+    checks: null
+    severity: HIGH
+  - id: 4.1.5
+    name: Ensure that default service accounts are not actively used. (Manual)
+    description: The default service account should not be used to ensure that rights granted to
+        applications can be more easily audited and reviewed.
+    checks: null
+    severity: HIGH
+  - id: 4.1.6
+    name: Ensure that Service Account Tokens are only mounted where necessary (Manual)
+    description: |
+      Service accounts tokens should not be mounted in pods except where the workload
+      running in the pod explicitly needs to communicate with the API server
+    checks: null
+    severity: HIGH
+  - id: 4.1.7
+    name: Avoid use of system:masters group (Manual)
+    description: |
+      The special group system:masters should not be used to grant permissions to any user
+      or service account, except where strictly necessary (e.g. bootstrapping access prior to
+      RBAC being fully available)
+    checks: null
+    severity: CRITICAL
+  - id: 4.1.8
+    name: Limit use of the Bind, Impersonate and Escalate permissions in the Kubernetes cluster (Manual)
+    description: |
+      The special group system:masters should not be used to grant permissions to any user
+      or service account, except where strictly necessary (e.g. bootstrapping access prior to
+      RBAC being fully available)
+    checks: null
+    severity: CRITICAL
+  - id: 4.2.1
+    name: Minimize the admission of privileged containers (Manual)
+    description: Do not generally permit containers to be run with the securityContext.privileged flag set to true.
+    checks:
+      - id: AVD-KSV-0017
+    severity: HIGH
+  - id: 4.2.2
+    name: Minimize the admission of containers wishing to share the host process ID namespace (Manual)
+    description: Do not generally permit containers to be run with the hostPID flag set to true.
+    checks:
+      - id: AVD-KSV-0010
```

Comment:
This check is marked as manual however it has automatic checks

Reply:
ah that is part of the name in the CIS benchmarks... I guess in our case since we have automated it, I should remove it